### PR TITLE
fix deprecation warning

### PR DESCRIPTION
**Pull request by Flundrahn**
## Overview: What does this pull request change?

Fix issue #3178 Warning when using ParametricFunction with use_vectorized=True 

## Motivation and Explanation: Why and how do your changes improve the library?
For future Numpy compatibility.

## Further Information and Comments
See details in the issue report.

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: manim/mobject/graphing/functions.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def generate_points(self):
                 x, y, z = self.function(t_range)
                 if not isinstance(z, np.ndarray):
                     z = np.zeros_like(x)
-                points = np.stack(zip(x, y, z), axis=0)
+                points = np.stack([x, y, z], axis=1)
```
Comment (naveen521kk) [inline manim/mobject/graphing/functions.py:161]: Any reason why you changed the `axis` parameter here?

Comment (Flundrahn) [reply manim/mobject/graphing/functions.py:161]: The list does not return the same thing as the zip, one have to flip the axis for the same output.

But! Don't take my word for it, in the related Issue there is a code snippet you can put in a Scene, debug and test. If you are using VS Code I have a launch.json debug config you can use.

Comment (Flundrahn) [reply manim/mobject/graphing/functions.py:161]: Here
```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Python: Current File",
            "type": "python",
            "request": "launch",
            "program": "manim.py ${file}",
            "console": "integratedTerminal"
        }
    ]
}
```

Comment (behackl) [reply manim/mobject/graphing/functions.py:161]: Tested with a small example,
```pycon
>>> x = np.linspace(0, 5, 10)
>>> y = x**2
>>> z = x*0.1 + 1
>>> np.all(np.stack(zip(x, y, z), axis=0) == np.stack([x, y, z], axis=1))
True
```

**Review by behackl — APPROVED** [review]
LGTM

File: manim/mobject/graphing/functions.py (hunk 1 above)
Comment (naveen521kk) [reply manim/mobject/graphing/functions.py:161]: Oh right, I didn't notice that. Thanks!

**Review by naveen521kk — APPROVED** [review]
lgtm!

**Pull request merged**